src/server/config-example.json

```diff
@@ -1,143 +0,0 @@
-{
-  "organization": "RocketChat",
-  "organizationHomepage": "https://rocket.chat/",
-  "organizationGithubUrl": "https://github.com/RocketChat",
-  "authToken": "",
-  "adminPassword": "123456",
-  "delay": "10",
-  "serverPort": "62050",
-  "contributors": [
-    "HusseinElFeky",
-    "rodriq",
-    "lolimay",
-    "divyanshub024",
-    "knrt10",
-    "govinddixit",
-    "shubhsherl",
-    "fliptrail",
-    "ubarsaiyan",
-    "snuff4",
-    "thayannevls",
-    "dev-ritik",
-    "nbamir",
-    "prajvalraval",
-    "thedreamsaver",
-    "bhardwajaditya",
-    "davide95",
-    "codekhal",
-    "pranavpandey1998official",
-    "kareemhamdy100",
-    "kailash0311",
-    "auraofdivinity",
-    "arthurbdiniz",
-    "cool-fire",
-    "pranay414",
-    "ashwaniydv",
-    "savish28",
-    "supra08",
-    "thelimitbreaker",
-    "eliezer-of",
-    "anhanh11001",
-    "vankineenitawrun",
-    "bodasadalla98",
-    "aryankoul",
-    "aditya-kolla",
-    "sudipt1999",
-    "shloak17107",
-    "djorkaeffalexandre",
-    "adizbek",
-    "avsingh999",
-    "mushroomgenie",
-    "gsunit",
-    "pratyush1197",
-    "promehul",
-    "prateekguptaiiitk",
-    "sk1410",
-    "alinasser96",
-    "dtran26",
-    "alexfanchina",
-    "abrahamks",
-    "rngo1214",
-    "jaya738",
-    "bolariin",
-    "gargarchit",
-    "shubhambansal1997",
-    "geka412315",
-    "vishal260700",
-    "hv7214",
-    "ilovrencic",
-    "leoking-crey",
-    "malivp3494",
-    "boredomed",
-    "mehtabhazra1998",
-    "anshulmak",
-    "smithshelke",
-    "saran-shabd",
-    "mohan122",
-    "mercuretony",
-    "jaysheombar",
-    "gaurav-punjabi",
-    "vanntile",
-    "vatsalkul",
-    "poppy22",
-    "hyousi",
-    "tummler13",
-    "tusharjindal353",
-    "iswiftshek",
-    "resuls",
-    "b3u",
-    "rahul-vijay",
-    "aditodkar",
-    "imshubhamsingh",
-    "zarbielli",
-    "aswinmprabhu",
-    "ayush517",
-    "kgomez0110",
-    "pedro12909",
-    "kanishk98",
-    "ani4aniket",
-    "apooravc",
-    "feat7",
-    "vansjyo",
-    "jervis446",
-    "khalea",
-    "rohithgoudm",
-    "arora-yash",
-    "akash491",
-    "aymanazzam",
-    "paritosh9199",
-    "saurabhdaware",
-    "aloksingh3112",
-    "adityayadavalli1",
-    "devpool007",
-    "ahmednader42",
-    "manveerbasra",
-    "adrianrobotka",
-    "omarghoneim",
-    "chetat",
-    "bhanu256",
-    "erscheinung",
-    "water2015sit",
-    "look4yasharth",
-    "mozammilkhan",
-    "mohamedar97",
-    "matheusmiranda",
-    "real-player",
-    "sakshi84250",
-    "khalifa1997",
-    "snathjr",
-    "mayank2424",
-    "nuclearpeanutt",
-    "gautime",
-    "jongoodrum",
-    "huycong151197",
-    "shorpee",
-    "ilarionhalushka",
-    "rkaushik15",
-    "seenivasanseeni",
-    "singh728om",
-    "ahmad-ali-sherazi",
-    "abdelazeem777\t"
-  ],
-  "startDate": "1970-01-15"
-}
```

Comment:
@Sumitraj387 You don't need to delete the `config-example.json` file. It's a crucial part of getting the config.json file initialized.